src/Actuators/Slack/SlackActuator.php

```diff
@@ -37,17 +38,19 @@ public function __construct()
      * The SlackActuator determines the type of Slack message so as to call the apporpriate Slack API endpoint.
      *
      * @param $action
-     * @param array $arguments
+     * @param Map $arguments
      * @return mixed
      */
-    public function perform(string $action, $arguments = [])
+    public function perform(string $action, Map $arguments = null)
```

Comment:
An action may not have any arguments. I've modified the code to have a try / catch block which also makes the code a bit simpler. Removed the check on the type of action since the actuator only supports one and the Agent container already does a check.